diff -p -up monkey-bubble-0.4.0/src/ui/ui-main.c~ monkey-bubble-0.4.0/src/ui/ui-main.c
--- monkey-bubble-0.4.0/src/ui/ui-main.c~	2006-08-01 15:47:04.000000000 +0200
+++ monkey-bubble-0.4.0/src/ui/ui-main.c	2009-05-21 11:03:07.000000000 +0200
@@ -668,7 +668,7 @@ static void show_error_dialog (GtkWindow
                                          GTK_DIALOG_DESTROY_WITH_PARENT,
                                          GTK_MESSAGE_ERROR,
                                          GTK_BUTTONS_CLOSE,
-                                         message);
+                                         "%s", message);
     
         g_signal_connect (G_OBJECT (dialog), "response", G_CALLBACK (gtk_widget_destroy), NULL);
     
